content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+
+## NFTs on Solana
+
+A Solana NFT is a non-divisible token with associated metadata that comes from a token mint with a maximum supply of 1. Let's break that down a bit. An NFT is a standard token from the Token Program. What makes it unique is that it:
+
+1. Has 0 decimals so that it cannot be divided into parts
+2. Comes from a token mint with supply of 1 so that only 1 of these tokens exists
+3. Comes from a token mint whose authority is set to null (to ensure that the supply never changes)
+4. Has an associated account that stores metadata
+
+While the first three points are standard features that can be achieved with the SPL Token Program, the associated metadata requires some additional functionality.
+
+Typically, an NFT’s metadata has both an on-chain and off-chain component. The on-chain metadata is stored in an account associated with the token mint and contains a URI attribute that points to an off-chain JSON file. The off-chain component stores additional data and a link to the image. Permanent data storage systems such as Arweave are often used to store the off-chain component of NFT metadata.
+
+Below is an example of the relationship between on-chain and off-chain metadata. The on-chain metadata contains a URI field that points to an off-chain `.json` file that stores the link to the image of the NFT and additional metadata.
+![Screenshot of Metadata](../assets/solana-nft-metaplex-metadata.png)
+
+## Metaplex
+
+The defacto standard tools for NFTs on Solana are provided by an organization called Metaplex. Metaplex is an organization that provides a suite of tools that simplify the creation and distribution of NFTs on the Solana blockchain. 
+
+One of the core programs provided by Metaplex is the Token Metadata Program. The Token Metadata Program standardizes the process of associating metadata to an NFT. When you create an NFT with Metaplex, the Token Metadata Program creates a metadata account using a Program Derived Address (PDA) with the token mint as a seed. This allows the metadata account for any NFT to be located deterministically using the address of the token mint.
+
+In addition to the Token Metadata PRogram, Metaplex provides Candy Machine v2. Candy Machine v2 leverages the Token Metadata Program to create tokens, manage their metadata, and customize their distribution.
+
+## Create an NFT with Metaplex
+
+Candy Machine v2 simplifies the process of creating an NFT so you can focus on what's important: the NFT metadata and assets. Creating the NFT involves:
+
+1. Preparing assets and metadata
+2. Configuring your candy machine
+3. Uploading your candy machine
+
+### Prepare assets and metadata
+
+Before creating a collection you need to prepare the assets and metadata for your collection. The metadata is simply a JSON file that includes the following attributes:
+
+- `name` - the name of the NFT that will display on-chain.
```

Comment:
@jamesrp13 sugg: remove ending punctuation from list items (sometimes it makes sense if the item is a couple sentences, but in general, they're not needed